bundles/automation/org.eclipse.smarthome.automation.sample.java.demo/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: Eclipse SmartHome Automation Java Demo
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-SymbolicName: org.eclipse.smarthome.automation.sample.java.demo
-Bundle-Vendor: Bosch Software Innovations GmbH
+Bundle-Vendor: Eclipse.org/SmartHome

```

Comment:
@kaikreuzer what is the arrangement with Bosch? Can we change to Eclipse.org here?

Reply:
Yes, certainly - that was probably just by accident.